pyproject.toml

```diff
@@ -35,6 +35,7 @@ scipy = ">=1.4.1"
 seaborn = ">=0.11.2"
 torch = ">=1.9"
 torchmetrics = ">=0.5.1"
+yolov5 = "^7.0.9"
```

Comment:
did you run `poetry update` or just `poetry add yolo5`? and in the last PR?

Reply:
I ran `poetry update` in the previous PR, which unfortunately seem to have broken the `case studies` workflow. 

In this PR I just ran `poetry add yolov5`. We don't really use a lot of yolov5, so I also could have copied/pasted some of the yolov5 code into the bliss repo. We may want to do that in the future so it's easier to modify.